QueueManagerTest/QueueManagerTests.cs

```diff
@@ -1,20 +1,46 @@
 ﻿using System.Collections.Generic;
 using NUnit.Framework;
 using QueueManager;
+using System.IO;
+using System;  
 
 namespace QueueManagerTest
 {
     [TestFixture]
     public class QueueManagerTests
     {
         QueueManagers _queueManager;
+        string _pathString;
+        string _upperLevelFolderName;
+        string _testFileName;
+        TextWriter _testFile;
+        string _pathWithFile;
+        List<string> _linesInFile;
 
         [SetUp]
         public void SetUp()
         {
             _queueManager = new QueueManagers();
+            _upperLevelFolderName = @"..\..\Test Files\";
+            _pathString = Path.GetFullPath(_upperLevelFolderName);
```

Comment:
The setup creates this file and directory, and if I'm not mistaken the teardown removes the directory and its contents. Therefore, there shouldn't ever be a case that the directory or the files exist in the setup. If they do something fishy is happening and should be looked at. The exists checks here should be redundant and can go.